contracts/Bridge.sol

```diff
@@ -47,10 +47,9 @@ contract Bridge {
     event RelayerThresholdProposalVote(Vote vote);
     event RelayerThresholdChanged(uint indexed newThreshold);
     event Deposit(
-        uint256 indexed originChainID,
         uint256 indexed destinationChainID,
         address indexed originChainHandlerAddress,
-        uint256         depositNonce
+        uint256 indexed depositNonce
```

Comment:
We can only index a maximum of 3 parameters of an event, I don't think we're able to access depositNonce as a topic in this case though the indexed keyword is specified.